app/src/main/res/menu/drawer_view.xml

```diff
@@ -7,12 +7,12 @@
         <item
             android:id="@+id/nav_registration"
             android:title="@string/update_registration" />
-        <item
-            android:id="@+id/nav_more_templates"
-            android:title="@string/more_templates" />
         <item
             android:id="@+id/nav_bloom_templates"
             android:title="@string/bloom_templates" />
+        <item
+            android:id="@+id/nav_more_templates"
```

Comment:
If you change the order of menu items here, you will also need to update: global_menu_array.xml and DrawerItemClickListener.kt

Reply:
DrawerItemClickListener.kt also needs fixing to add the bloom lib download action